frontend-ssr/components/feedback-review/CommentBlock.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Link from 'next/link';
+import { Comment } from './comment.model';
+import styles from './comment-block.module.scss';
```

Comment:
Let's use the following naming for the styling files `CommentBlock.module.scss`.

Reply:
Done